### DISCUSSION TITLE
Mitosis: Future or Failure

### DISCUSSION THREAD

**Discussion by mees-van-wel** [mantinedev/mantine#4302] [Feature requests]
I'm switching from Next.js to Qwik due to resumability and already miss Mantine big time.
On my journey to find a components library for which (which ended quickly as there currently are none), I came across [Mitosis](https://github.com/BuilderIO/mitosis). 

I understand that Mantine is a React only components library but I think like Qwik with it's resumability, Mitosis will be the future of components libraries. I'm not asking to directly do something with this, I mean it's even still in beta but it might be nice to just check it out and keep it in the back of our heads. Maybe some day the time will come that Mantine can truely be **"THE ONE"** components library.

**Comment by rtivital** [comment]
I'm building Mantine to use it in my work and personal projects. I use React with Next.js pages router and Vite for my applications. I do not see any benefit or a reason to maintain libraries for other frameworks. It takes significant effort and gives nothing in return.
